README.md

```diff
@@ -492,6 +492,15 @@ To launch tests run:
     And with integration tests:
     GO111MODULE=on go test ./... --tags integ
 
+# Sentry integration
+
+[Sentry](https://sentry.io) is used to capture crash reports for `dnode` and `dncli` applications.
+By default the integration is switched off.
+
+The following environment variables should be defined in order to enable the integration:
+* `SENTRY_DSN` - Sentry DSN token (`https://[token]@sentry.io/5167345`);
+* `SENTRY_ENVIRONMENT` - sets the environment code to separate events from testnet and production (could be empty);
```

Comment:
In the DVM we'r using the [`DVM_SENTRY_DSN`](https://github.com/dfinance/dvm/blob/master/src/cli/config.rs#L9) env var. Maybe there might be the same or something like `DNODE_SENTRY_DSN`?

Reply:
Not sure as those are standard evns for Sentry. @borispovod , @papa-stiflera what's you opinion about that? If we gonna rename them, it should be `DN_SENTRY_DSN` (using already used prefix)